README.md

```diff
@@ -10,20 +10,21 @@ based on [OpenLink Virtuoso](https://virtuoso.openlinksw.com).
 
 The following RDF triples will be imported once the service starts:
 
-- [Corporate body classification](https://publications.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/corporate-body-classification/version-20180926-0)
-- [Corporate body](https://op.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/corporate-body/version-20200624-0)
-- [Country](https://op.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/country/version-20200624-0)
-- [Digital Europa Thesaurus](https://op.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/det/version-1.2.0)
-- [EuroVoc Thesaurus](https://op.europa.eu/en/web/eu-vocabularies/th-dataset/-/resource/dataset/eurovoc/version-20200630-0)
-- [EU Programme](https://op.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/eu-programme/version-20200624-0)
-- [Language](https://publications.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/language/version-20200624-0)
-- [Organization type](https://publications.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/organization-type/version-20170920-0)
-- [Place](https://op.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/place/version-20200624-0)
-- [Public event type](https://publications.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/public-event-type/version-20180926-0)
-- [Resource type](https://op.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/resource-type/version-20200624-0)
-- [Target audience](https://publications.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/target-audience/version-20180620-0)
-- [Knowledge4Policy list](https://op.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/k4p-list)
-- [Sustainable Development Goals](https://op.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/sdg)
+- [Corporate body classification](https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/corporate-body-classification&version=20180926-0)
+- [Corporate body](https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/corporate-body&version=20210116-0)
```

Comment:
All the urls in the form `https://op.europa.eu/en/web/eu-vocabularies/concept-scheme` are wrong, we should be pointing at the main page with the information and not the table with the list of terms. To pick the correct page, go to the main authority tables page https://op.europa.eu/en/web/eu-vocabularies/authority-tables and find the table there. Then click the name of it, it will bring to a page with the url that starts with `https://op.europa.eu/en/web/eu-vocabularies/dataset/-/resource?`. 
Then, go to the correct version of the previous link. For example corporate body version was 20200624 so we need to put a link to the same version.

Reply:
Fixed